packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -165,7 +165,7 @@ onDestroy(() => {
 });
 
 function gotoBuildImage(): void {
-  router.goto('/images/build');
+  router.goto('/images/build?taskId=');
```

Comment:
you should not need to specify the taskId parameter here, not having it makes it undefined